src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -58,6 +58,45 @@
  * <pre>
  * &lt;module name=&quot;SingleLineJavadoc&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *   public class Test {
+ *     /** &#64;return integer value *&#47; // violation
+ *     // Javadoc has one at-clause, should be written in multiple lines
+ *     public int sum(int a, int b) {
+ *       return a + b;
+ *     }
+ *   }
+ *   public class Test {
+ *     /**
+ *      * &#64;return integer value
+ *      *&#47; // ok
+ *     public int sum(int a, int b) {
+ *       return a + b;
+ *     }
+ *   }
+ * </pre>
+ * <p>
+ * To configure this check to print violations for unclosed HTML tags.
```

Comment:
1. print -> report
2. dot at the end -> colon

Reply:
ok